multiaccounts/accounts/database.go

```diff
@@ -902,6 +894,7 @@ func (db *Database) SaveOrUpdateAccounts(accounts []*Account, updateKeypairClock
 	return err
 }
 
+// This function doesn't save/update keycards.
```

Comment:
Although it probably makes sense in terms of this PR to have a comment like this, I would prefer to state the obvious at first and write what it actually does (just to emphasise why mentioning of what it doesn't do is important) because this function doesn't do an infinite number of things :)

Definitely not a deal breaker

Reply:
Comment updated. ;)